src/batch-client/index.ts

```diff
@@ -306,14 +307,29 @@ export class ZimbraBatchClient {
 			name: type,
 			body: {
 				action: {
-					id: id || (ids || []).join(','),
+					id: id || [ids].join(','),
```

Comment:
do we need two params ? id and ids?

Reply:
Many of the actions are passing ids or id, this part is been use in msgaction, Folderaction itemaction so I haven't checked those mutation

Reply:
<img width="803" alt="Screen Shot 2020-10-06 at 7 03 21 AM" src="https://user-images.githubusercontent.com/33997265/95148936-3af28200-07a2-11eb-9ff5-b89918190ed9.png">
just a example

Reply:
ok